src/App.tsx

```diff
@@ -1,7 +1,35 @@
-import React from 'react';
+import { Component } from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  previous: string,
+};
+
+export class App extends Component<{}, State> {
+  state = {
+    previous: '',
```

Comment:
why is it called previous? what does it represent? it should be named "key" or something like that